dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -6509,11 +6522,14 @@ definitions:
         position: 12
         description: "The id of the associated checker workflow"
         readOnly: true
-      conceptDoi:
-        type: "string"
-        position: 13
-        description: "The Digital Object Identifier (DOI) representing all of the\
-          \ versions of your workflow"
+      inputFileFormats:
```

Comment:
same here, is there a reason to change the interface when we're just changing the back-end laziness of it?